otherlibs/systhreads/st_stubs.c

```diff
@@ -716,27 +768,22 @@ CAMLprim value caml_thread_exit(value unit)   /* ML */
   } else {
     /* Bytecode, or thread created from C */
     st_thread_exit();
-  }
+  };
```

Comment:
Why ?

Reply:
Indeed!
Fixed in https://github.com/ocaml-multicore/ocaml/commit/32e8f0f0a6055da5bf1330a69d62756f48ac214e